public/Algorithm/BinaryTree/claude sonnet 4.6 extended/105. Construct Binary Tree from Preorder and Inorder Traversal/README_React.html

```
@@ -0,0 +1,2071 @@
+from typing import Optional
+
+# 再帰深度の上限を緩和（デフォルト1000 → 偏った木で3000段になりうる）
+sys.setrecursionlimit(10_000)
+
+
+class Solution:
+    def buildTree(
+        self,
+        preorder: list[int],
+        inorder: list[int],
+    ) -> Optional[TreeNode]:
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)